Changelog.md

```diff
@@ -19,6 +19,8 @@
 ## [v2.3.3]
 - Fix bug where uploading scanned exam pages with overwriting option selected did not update submission files (#6768)
 - Fix bug: "Download Submissions" download link was not being rendered from partial view (#6779)
+- Bug Fix: Prevent assigning inactive graders to a group, throwing a red flash message (through ```graders_controller```)
```

Comment:
This entry should be added to the "unreleased" section. Also you don't need to say the second part, just say the "Prevent assigning inactive graders to a group".